tests/integration_tests/net/test_dhcp.py

```diff
@@ -46,12 +46,20 @@ def test_noble_and_newer_uses_dhcpcd_by_default(self, client):
         CURRENT_RELEASE < NOBLE,
         reason="earlier Ubuntu releases have a package named dhcpcd5",
     )
-    def test_noble_and_newer_force_dhcp(self, client):
+    @pytest.mark.parametrize(
+        "client, package",
```

Comment:
Won't this override the client fixture? I think we need a different name than `client`.